### DISCUSSION TITLE
Create a common set of tags to be used as categorization for packages

### DISCUSSION THREAD

**Discussion by AdmiringWorm** [chocolatey-community/chocolatey-packages#1814] [Ideas]
Currently we have a common set of tags that need to be used in the metadata for packages located in this repository (when appropriate).

However none of these tags are part of any way to actually categorize the tags that can be used by users to search by (ie, browser for web browsers, editor for text editors, etc).

As such, I believe it would be useful to include in our guidelines of a set of tags that we need to use as a category where the package lies in, at least one of the tags we agree on would need to be used for every package that gets updated/added in the repository.

This suggestion initially started a few years ago as part of documentation on docs.chocolatey.org, but as a start it is useful for it to be possibly started here in this repository as many use it as a template when creating their own repository.

Some suggestion that have come forth as categorization tags are:

- productivity
- programming
- utility
- browser
- drivers
- server
- client
- addon

Additionally I would also recommend the following tags be added as categorization tags we are to use:

- editor
- game
- web
- plugin/addon (whichever is appropriate)
- extension (only for Chocolatey extension packages) (_These are meant to eventually be moved out of this repository, so may not make sense to have this tag in the Guidelines_)

I have not included the tags `admin`, `trial`, `notsilent` and `license` tags as these are not categorization tags, but useful in other ways instead.

I welcome other suggestions as well, and do not see these categorization tags as something that is completely final even when implemented.

The tags to use would also need to be documented in our guidelines, and enforced by anyone reviewing PR's.

Reference to link on the documentation site: https://github.com/chocolatey/docs/issues/343

/cc @chocolatey-community/community-moderators

**Comment by mwallner** [comment]
I like it, though I'd also like to see tags such as "portable" or "installer" as well as "meta" (= 'this package only has dependencies to other packages but does not include any logic itself'). 
I know it doesn't _really_ fit in there, but given that the IDs are permanent and I've seen a couple of packages that changed their install behavior (i.e. msi to being portable or meta only), I wouldn't rely on the package ID to give the clue, rather than having tags (which could possibly change from one version to another).

**Reply by AdmiringWorm** [reply]
Not a bad idea, not sure about the `meta` tag but a `installer`/`portable` tag would make sense I think, although not as a categorization tag but definitely something that would be worthwhile to investigate to see if we want to add those two to our guidelines as well.

**Reply by JPRuskin** [reply]
On a similar (but not really matching the intent of this discussion) note, I would be interested in an "LTS" tag - but I'm happy to raise that in a completely different area.

**Reply by AdmiringWorm** [reply]
@JPRuskin I think that may need a bigger discussion about it (mostly on what to use for that).

Not sure if an `LTS` tag would be the best (it would actually be `lts` since tags should be lowercase).

Mind opening a new discussion for labelling lts releases?

**Comment by AdmiringWorm** [comment]
I'll leave this discussion open until the end of the month, if there are no other suggestions or disagreement with the tags listed I'll get an issue created to update the Guidelines that one of the tags mentioned in the initial comment need to be present on each package located in the repository.

While I did not mention this initially, I am open to hearing the thoughts of people not normally working on this repository.

**Reply by AdmiringWorm** [reply]
I bit more than a month went by, but an issue was finally added based on this discussion
